pkg/fuse/device_linux.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * JuiceFS, Copyright 2023 Juicedata, Inc.
```

Comment:
Do we copy this file from Chaos Mesh? please keep the original license header

Reply:
Actually, this file existed in the early version of Chaos Mesh, it has been removed for several years. Should we still keep the original license header?

Reply:
yes, if it's released once

Reply:
Ok, kept.